addons/models/auth/AuthPolicy.json

```diff
@@ -0,0 +1,394 @@
+          "value": "allow",
+          "ordinal": 0
+        },
+        {
+          "value": "deny",
+          "ordinal": 1
+        },
+        {
+          "value": "allowExceptions",
+          "ordinal": 2
+        },
+        {
+          "value": "denyExceptions",
+          "ordinal": 3
+        },
+        {
+          "value": "dataMask",
+          "ordinal": 4
+        },
+        {
+          "value": "rowFilter",
+          "ordinal": 5
+        }
+      ]
+    },
+    {
+      "category": "ENUM",
+      "name": "AuthPolicyResourceCategory",
+      "description": "Policy resource category",
+      "serviceType": "atlas_core",
+      "typeVersion": "1.0",
+      "elementDefs":
+      [
+        {
+          "value": "ENTITY",
+          "ordinal": 0
+        },
+        {
+          "value": "RELATIONSHIP",
+          "ordinal": 1
+        },
+        {
+          "value": "TAG",
+          "ordinal": 2
+        },
+        {
+          "value": "CUSTOM",
+          "ordinal": 3
+        },
+        {
+          "value": "TYPEDEFS",
+          "ordinal": 4
+        },
+        {
+          "value": "ADMIN",
+          "ordinal": 5
+        }
+      ]
+    }
+  ],
+  "structDefs":
+  [
+    {
+      "name": "AuthPolicyValiditySchedule",
+      "description": "Validity schedule struct for policy",
+      "serviceType": "atlan",
+      "typeVersion": "1.0",
+      "attributeDefs":
+      [
+        {
+          "name": "policyValidityScheduleStartTime",
+          "typeName": "string",
+          "indexType": "STRING",
+          "cardinality": "SINGLE",
+          "isIndexable": false,
+          "isOptional": false,
+          "isUnique": false
+        },
+        {
+          "name": "policyValidityScheduleEndTime",
+          "typeName": "string",
+          "indexType": "STRING",
+          "cardinality": "SINGLE",
+          "isIndexable": false,
+          "isOptional": false,
+          "isUnique": false
+        },
+        {
+          "name": "policyValidityScheduleTimezone",
+          "typeName": "string",
+          "indexType": "STRING",
+          "cardinality": "SINGLE",
+          "isIndexable": false,
+          "isOptional": false,
+          "isUnique": false
+        }
+      ]
+    },
+    {
+      "name": "AuthPolicyCondition",
+      "description": "Policy condition schedule struct",
+      "serviceType": "atlan",
+      "typeVersion": "1.0",
+      "attributeDefs":
+      [
+        {
+          "name": "policyConditionType",
+          "typeName": "string",
+          "cardinality": "SINGLE",
+          "isIndexable": false,
+          "isOptional": false,
+          "isUnique": false
+        },
+        {
+          "name": "policyConditionValues",
+          "typeName": "array<string>",
+          "cardinality": "SINGLE",
+          "isIndexable": false,
+          "isOptional": false,
+          "isUnique": false
+        }
+      ]
+    }
+  ],
+  "entityDefs":
+  [
+    {
+      "name": "AuthPolicy",
```

Comment:
Let's add Asset as a superType

Reply:
only possible if we also move Asset type creation here, till this point only base typeDefs are created